modules/gdscript/gd_parser.cpp

```diff
@@ -3965,11 +3960,11 @@ void GDParser::_parse_class(ClassNode *p_class) {
 
 					if (tokenizer->get_token() != GDTokenizer::TK_COMMA) {
 						//just comma means using only getter
-						if (tokenizer->get_token() != GDTokenizer::TK_IDENTIFIER) {
-							_set_error("Expected identifier for setter function after 'notify'.");
+						if (!tokenizer->is_token_literal()) {
+							_set_error("Expected literal for setter function after 'setget'.");
```

Comment:
Old relict? 😄 This is more descriptive, cool. But I'd still change it to back to identifier.